src/lesson5.java

```diff
@@ -0,0 +1,21 @@
+public class lesson5 {
+    public static void main (String[]args){
+       employee Petrov = new employee("Petrov Serge","Division director","petrov@gmail.ru","+79376803322",200000,55);
+       Petrov.PrintPersonalData();
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.